.github/CODEOWNERS

```diff
@@ -389,11 +389,7 @@
 
 # Speed Improvement team.
 /app.yaml @vojtechjelinek
-/core/templates/dev/head/base_components/base_content_directive.html @jamesjay4199
-/core/templates/dev/head/base_components/BaseContentDirective.ts @jamesjay4199
-/core/templates/dev/head/base_components/header.html @jamesjay4199
-/core/templates/dev/head/base_components/warning_loader_directive.html @jamesjay4199
-/core/templates/dev/head/base_components/WarningLoaderDirective.ts @jamesjay4199
+/core/templates/dev/head/base_components/* @jamesjay4199 @vojtechjelinek
```

Comment:
I don't think you need the * at the end.

Reply:
Done